frontend/package.json

```diff
@@ -8,7 +8,7 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
-    "build": "ng build --prod",
+    "build": "ng build --prod=false",
```

Comment:
Instead of hardcoding the dev environment, I'd suggest adding another task (here and in the backend's gradle): `gradle buildDev` and `gradle buildProd` for example which will build either the dev or the prod environment. What do you say?

Reply:
Interestingly, the build fails in when --prod is true, because the apiUrl is not set in the prod environment. Have I missed something?

Reply:
This is somewhat important. Can we set this to a sensible default value?

Reply:
How about you invest some more time (say half an hour), and if it doesn't work, it's a separate issue?